### PR TITLE
fix: button layout and margins in storybook

### PR DESCRIPTION
**Pull request by doichev-kostia**

### CODE REVIEW

File: stories/buttons/index.jsx [hunk 1]
```diff
@@ -11,19 +19,49 @@ import s from './styles.scss';
...
+              <Button style={ buttonStyles } className={ s.mb1 }>Text</Button>
+            </div>
+          </Col>
+          <Col xs='4' m='4' l='4' style={ buttonCellStyles }>
+            <div style={ {
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
+            } }>
+              <Button style={ buttonStyles } className={ s.mb1 }
```
Comment (vlacher12) [inline stories/buttons/index.jsx:50]: fix eslint, pls

File: stories/buttons/styles.scss [hunk 2]
```diff
@@ -3,3 +3,11 @@
 .secondaryButton {
   background-color: blue;
 }
+
+.mb1{
```
Comment (vlacher12) [inline stories/buttons/styles.scss:7]: stylelint

**Comment by doichev-kostia** [comment]
@vlacher12, fixed

**Comment by vlacher12** [comment]
@doichev-kostia fix conflicts pls

**Comment by doichev-kostia** [comment]
@vlacher12✅

**Comment by lazzystep** [comment]
Closing due to 0 changes in it

**Pull request closed without merging**